packages/gatsby-link/package.json

```diff
@@ -26,7 +26,7 @@
   "license": "MIT",
   "main": "index.js",
   "peerDependencies": {
-    "gatsby": ">2.0.0-alpha"
+    "gatsby": "^2.0.0-alpha"
```

Comment:
As @KyleAMathews said, let's just move (all of these) to `^2.0.0` and we can get this merged.